extensions/blocks/podcast-player/edit.js

```diff
@@ -266,7 +257,29 @@ const PodcastPlayerEdit = ( {
 	return (
 		<>
 			<BlockControls>
-				<Toolbar controls={ toolbarControls } />
+				{ /* @todo Fallback can be removed when WP 5.4 is the minimum supported version. */ }
```

Comment:
minor suggestion: make it sense moving all logic into a variable or a component?

```es6
<PodcatsPlayerToolbar />
```

and deal with this there. Please feel free to ignore it.

Reply:
I'd rather keep it modular so it can be cleaned up once WordPress 5.5 was released.